code/index.html

```diff
@@ -1,30 +1,83 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Quicksand&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Dancing+Script&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <!-- video or image as a header is cool :) -->
+  <header>
+    <h1>Nordic Dance Art</h1>
+  </header>
+
+  <!--Poem beneath the header-->
+  <!--<div>
+    <h2>Life isn´t about waiting for the storm to pass...</h2>
+    <h2>It´s about learning to Dance in the rain</h2>
+  </div>-->
```

Comment:
Code that we don't use can be deleted before the pull request to make it "clean". If you want to save the code for later you can make a commit with it and after delete it. Then you can go back to that commit later on to get the code!